tests/console/consoletest_setup.pm

```diff
@@ -11,47 +11,64 @@
 # Summary: console test pre setup, performing actions required to run tests
 # which are supposed to be reverted e.g. stoping and disabling packagekit and so on
 # Permanent changes are now executed in system_prepare module
-# - Save screenshot
-# - Disable mail notifications
+# - Setup passwordless & questionless ssh login to localhost 127.0.0.1 ::1
 # - Disable/stop serial-getty service
+# - Disable mail notifications system-wide
+# - Enable pipefail system-wide
 # - Disable/stop packagekit service
-# - Enable pipefail
+# - Check console font
+
 # Maintainer: Oliver Kurz <okurz@suse.de>
 
 use base "consoletest";
 use testapi;
 use utils qw(check_console_font disable_serial_getty);
-use Utils::Backends qw(has_ttys use_ssh_serial_console);
-use Utils::Systemd 'disable_and_stop_service';
+use Utils::Backends qw(has_ttys);
+use Utils::Systemd qw(disable_and_stop_service systemctl);
 use strict;
 use warnings;
 
-sub disable_bash_mail_notification {
-    assert_script_run "unset MAILCHECK >> ~/.bashrc";
-    assert_script_run "unset MAILCHECK";
-}
 
 sub run {
     my $self = shift;
-    # let's see how it looks at the beginning
-    save_screenshot;
-    check_var("BACKEND", "ipmi") ? use_ssh_serial_console : select_console 'root-console';
+    $self->select_serial_terminal;
+
+    systemctl('start sshd');
+
+    # generate ssh key and use same key for root and bernhard
```

Comment:
`bernhard` should not be hardcoded anywhere - it's `testapi::$username`.

Reply:
sure, I will change that